docs/source/impl-guide/normalization.rst

```diff
@@ -39,159 +39,101 @@ Certain insertion or deletion alleles may have ambiguous
 
 .. _normalization-diagram:
 
-.. list-table::
-     **VRS Justified Normalization** A demonstration of fully justifying an insertion allele.
-   :class: reece-wrap
-   :header-rows: 1
-   :widths: 40 15 20
-   :align: left
-
-   *  -  | Steps
-      -  | `start` and `end` (interbase)
-	 | and allele sequences
-      -  | Equivalent representations
-   *  -  0. Given allele ``S:g.5_6delinsCAGCA`` defined on reference sequence S=TCAGCAGCT
-      -  | (4,6)
-         | (“CA”, “CAGCA”)
-      -  .. math:: TCAG \Bigl[ \frac{CA}{CAGCA} \Bigr] GCT
-
-   *  -  1. Trimming
-
-            a. Remove suffix common to all alleles, if any, and update end position.
-            b. Remove prefix common to all alleles, if any, and update start position. 
-
-            **Note:**  This example shows removing C prefix and A suffix.
-            Equivalently in this case, CA prefix or CA suffix could be removed.
-      -  | (5,5)
-         | (“”, “AGC”)
-      -  .. math:: TCAGC \Bigl[ \frac{}{AGC} \Bigr] AGCT
-
-   *  -  2. & 3. These conditions are False.
-      -
-      -
-
-   *  -  4a. Roll Left
-
-            Begin with trimmed alleles from (1).
-
-            While the terminal base of all non-empty alleles equals the base
-            prior to the current position, circularly permute all alleles right
-            one step and move the start left one position.
-
-            Shown: The 4 incremental steps of rolling left.
-      -  | (1,1)
-         | (“”, “CAG”)
-      -  .. math::
-            TCAGC \Bigl[ \frac{}{AGC} \Bigr] AGCT \\
-            TCAG \Bigl[ \frac{}{CAG} \Bigr] CAGCT \\
-            TCA \Bigl[ \frac{}{GCA} \Bigr] GCAGCT \\
-            TC \Bigl[ \frac{}{AGC} \Bigr] AGCAGCT \\
-            T \Bigl[ \frac{}{CAG} \Bigr] CAGCAGCT \\
-            \Rightarrow left\_roll = 4
-
-   *  -  4b. Roll Right
-
-            Symmetric case of step 4a.
-      -  | (8,8)
-         | (“”, “AGC”)
-      -  .. math::
-            TCAGC \Bigl[ \frac{}{AGC} \Bigr] AGCT \\
-            TCAGCA \Bigl[ \frac{}{GCA} \Bigr] GCT \\
-            TCAGCAG \Bigl[ \frac{}{CAG} \Bigr] CT \\
-            TCAGCAGC \Bigl[ \frac{}{AGC} \Bigr] T \\
-            \Rightarrow right\_roll = 3
-
-   *  -  5. Update position and alleles to fully justify within region of ambiguity.
-
-            To each trimmed allele from (1), prepend the *left_roll*
-            preceding reference bases and append the *right_roll*
-            following reference bases (corresponding to the interbase
-            reference spans (1,5) and (5,8) respectively).
-
-            Decrement the start position by *left_roll*, and increment the end
-            position by *right_roll*.
-      -  | (1,8)
-         | (“CAGCAGC”,
-         | “CAGCAGCAGC”)
-      -  .. math::
-            T \Bigl[ \frac{CAGCAGC}{CAGCAGCAGC} \Bigr] T
+.. figure:: ../images/normalize.png
+
+    A demonstration of fully justifying an insertion allele.
+
+    Reproduced from [2]_
 
 **References**
```

Comment:
Could we use a consistent format for references, please? I can work on this if we can achieve consensus on the format to be used. I would prefer the style used by NIH.

Reply:
Agreed. Done. (I used the Paperpile default, with bold titles.) 6523e51